cea/technologies/supply_systems_database.py

```
@@ -7,6 +7,9 @@
 
 
 import pandas as pd
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)